schemas/bridge/SCHEMA.md

```diff
@@ -149,4 +149,20 @@ Snapshot of token level data.
 | total_fees_usd           | The fees collected in USD, during this given period.      | number |
 | chain_id                 | The standard chain ID where the token resides.            | number |
 
+### Incentive Claim Data
+
+Transactional data on user level incentives claimed data.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the claim.                               | number |
+| chain_id                 | The standard chain id.                                    | number |
+| transaction_hash         | The hash of the transaction.                              | string |
+| log_index                | The event log. For transactions that don't emit event, create arbitrary index starting from 0. | number |
+| user_address             | The address of the user who claimed the incentives.       | string |
+| claimed_token_address    | The smart contract address of the claimed token.          | string |
+| amount                   | The amount of the token claimed, decimal normalized.      | number |
+| amount_usd               | The amount of claimed tokens in USD.                      | number |
+| other_incentive_usd      | (Optional) Any incentives outside of the claimed token, in this transaction, summed up in USD terms. | number |
```

Comment:
not sure if we want the tx_signer here. for scenarios where 1 EOA control many multisigs for claims

Reply:
Yep, I can add tx signer and claim address